package.json

```diff
@@ -36,6 +36,71 @@
     ],
     "main": "./main.js",
     "contributes": {
+        "chatParticipants": [
+            {
+                "id": "ms-azuretools.azure-agent",
+                "name": "azure",
+                "fullName": "%chatParticipants.azure.fullName%",
+                "description": "GitHub Copilot for Azure",
+                "isSticky": false,
```

Comment:
I intentionally did not make it sticky, in case that messes up the switch from standin `@azure` to real `@azure`.